textblob_fr/_text.py

```diff
@@ -334,7 +334,7 @@ def _read(path, encoding="utf-8", comment=";;;"):
     if path:
         if type(path) in string_types and os.path.exists(path):
             # From file path.
-            f = open(path)
+            f = open(path, encoding="utf-8")
```

Comment:
`open` does not include the `encoding` argument in Python 2, so I think you'll need to use `codecs.open`.

Reply:
> `open` does not include the `encoding` argument in Python 2, so I think you'll need to use `codecs.open`.

What about
`f = open(path, encoding="utf-8")`